ta/crypto_perf/Android.mk

```diff
@@ -1,4 +1,4 @@
-LOCAL_PATH := $(call my-dir)
-
-local_module := e626662e-c0e2-485c-b8c8-09fbce6edf3d.ta
-include $(BUILD_OPTEE_MK)
+LOCAL_PATH := $(call my-dir)
+
+local_module := e626662e-c0e2-485c-b8c8-09fbce6edf3d.ta

```

Comment:
I believe the UUID should be changed. The new TA is not compatible with the old one, it accepts a different list of commands.